kcp/login.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+set -euo pipefail
+
+KUBECONFIG_KCP="${KUBECONFIG_KCP:-$HOME/.kube/kcp.config}"
+KUBECONFIG_KCP_ARGOCD="${KUBECONFIG_KCP_ARGOCD:-$HOME/.kube/kcp.argocd-manager.config}"
```

Comment:
KUBECONFIG_KCP_ARGOCD - Is this being used anywhere in the script?

Reply:
Explicitly line 164, but implicitly as well because of the inclusion in KUBECONFIG_MERGED.

Reply:
I think you're talking about line 164 and KUBECONFIG_MERGED in kcp/run.sh script. This one is defined in the kcp/login.sh script.